client/src/index.jsx

```diff
@@ -27,7 +27,7 @@ import axios from 'axios';
 window.axios = axios;
 
 const middleware = [thunk];
-//if (process.env.NODE_ENV !== 'production') middleware.push(logger);
```

Comment:
The other PR has the same change

Reply:
I keep having to turn it back on so I know what is going on:)

Reply:
why don't keep it on all the time on `dev`, it is doing what it's meant to do.

Reply:
Because the other guy kept turning it off:)

Reply:
For the other guy: the logger for redux is important in dev mode and should be used, instead of be turned it off (if you must, don't commit it), I'd suggest to learn how to use it effectively, the benefits are real.